draft-ietf-taps-interface.md

```diff
@@ -2380,11 +2383,27 @@ Default:
 
 This property specifies that a message should be sent and received as a single
 packet without network-layer fragmentation, if possible.
-Attempts to send a message with this property set to a size greater than the
+This only takes effect when used with transports supporting this functionality
+and running over IP version 4. When it does take effect, setting this property to
+true will cause the Don't Fragment bit to be set in the IP header, and
+attempts to send a message with this property set to a size greater than the
 transport's current estimate of its maximum packet size (`singularTransmissionMsgMaxLen`)
-will
-result in a `SendError`. When used with transports supporting this functionality
-and running over IP version 4, the Don't Fragment bit will be set.
+will result in a `SendError`.
+
+#### No Transport-Layer Fragmentation {#no-transport-fragmentation}
+
```

Comment:
We could call this "segmentation"?
- At least I would use "segmentation" here, to avoid the ever-returning argument about fragments being good/bad.

Reply:
Here I actually disagree: I don't think that "segment" is the word used by all transports? Seems rather TCP-specific to me...   IMO, there's a slight risk that "segmentation" is more cryptic to some readers than "transport-layer fragmentation", which is rather explicit about what exactly it is. Don't you think?

Reply:
I don't agree - but I also don't care on which words we use, as long as we separate the definition of terms.

Reply:
Ok - if I couldn't convince you, I feel that going with your view is the better choice. I'll adjust the text.

Reply:
Or... call it transport fragmentation, and explicitly say that it is about how a transport segments the data into network-layer packets.

Reply:
I went with "No Segmentation".